CDN/postpreinstall.sh

```diff
@@ -73,8 +70,10 @@ systemctl restart aria2
 
 reboot_device
 
-exit 0
+systemctl enable devmgmt
```

Comment:
can you please add some comments and explain why this has to moved here ?

Reply:
Done. Let me know if any edits are required.